src/test/fixtures/stores.js

```diff
@@ -27,13 +27,12 @@ export function storeWithProfile(profile?: Profile): Store {
 }
 
 export function storeWithSimpleProfile(): Store {
-  // FIXME: These samples need to be seperated by two spaces instead of one.
```

Comment:
I went over other `getProfileFromTextSamples` calls too and fixed the wrong ones. Interestingly after changing those, non of the tests failed. And that made me wonder why we have multiple samples for those tests in the first place, but didn't want to change those tests fundamentally.
Also when I'm writing a test with `getProfileFromTextSamples` function, I tend to copy paste from somewhere else and then change it for that specific test. So at least that will make the future tests less error prone from copy pasting(at least for me :))